packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -926,16 +927,19 @@ export default class Editor {
     /**
      * Converter for dark mode that runs all child elements of a node through the content transform function.
      * @param node The node containing HTML elements to convert.
+     * @param skipRootElement Optional parameter to skip the root element of the Node passed in, if applicable.
      */
-    private convertContentToDarkMode(node: Node): () => void {
+    private convertContentToDarkMode(node: Node, skipRootElement?: boolean): () => void {
         let childElements: HTMLElement[] = [];
 
         // Get a list of all the decendents of a node.
         // querySelectorAll doesn't return a live list when called on an HTMLElement
         // So we use getElementsByTagName instead for HTMLElement types.
         if (node instanceof HTMLElement) {
             childElements = Array.prototype.slice.call(node.getElementsByTagName('*'));
-            childElements.unshift(node);
+            if (!skipRootElement) {

```

Comment:
This is a style question mostly, but would folks prefer "includeRootElement" instead?

Reply:
In what case skipRootElement is false?

Reply:
Above, on line 183:
`const darkModeTransform = this.isDarkMode() ? this.convertContentToDarkMode(node) : null;`

Reply:
I see.